README.md

```diff
@@ -27,7 +27,11 @@ Download the latest release of the application [here]().
 
 ### Running your own servers
 
-For testing locally, you can use the docker images available in this repository.
+For testing locally, you can use the docker images available in this repository.  
+The docker images are hosted on *github packages* , (at the time of writing) so *you will need to log in first with your github account* (eventually authentication token) to pull those images (even for public repository):  
```

Comment:
The docker images are made available through *github packages* , which requires authentication to pull the any images (even for public repository). You will thus need need to log in first with your github account or use an authentication token to pull those images:

Reply:
Done